docs/pages/Troubleshooting.md

```diff
@@ -6,5 +6,9 @@ An HTTPS server or proxy that was previously running on a domain (e.g. `localhos
 #### Resolution
 You need to delete the HSTS mapping in [chrome://net-internals/#hsts](chrome://net-internals/#hsts) by entering the domain name (e.g. `localhost`) and pressing "delete".
 
+## UI5 Project
+
+
```

Comment:
Something to add here?

Reply:
Yes, I wanted to mention `ui5 tree` and `ui5 tree --full` for debugging simple dependency issues. But that's a topic for another PR.

I removed the headline.